teps/0098-workflows.md

```diff
@@ -125,62 +54,112 @@ simplify the Tekton experience. It can provide a single place to declare all the
 
 ### Goals
 
-<!--
-List the specific goals of the TEP.  What is it trying to achieve?  How will we
-know that this has succeeded?
--->
-
 * Provide a way to describe entire CI/CD configuration (not just pipeline definition) in a single place
 * Make managing Tekton configuration less verbose and repetitive.
-* Support end to end CI workflows from source control
 * Make it easier and simpler for end users to get started with a pure Tekton CI system
-* Allow clear separation of concerns between platform developers and app teams
-* Allow for clear separation between run configuration and pipeline definition?
+* Allow clear separation of concerns between platform developers and app teams,
+including making it easier to separate notifications and other infrastructure out of the Pipelines being run
+* Create a Workflows conformance spec
```

Comment:
These are new to me, but both sound good!

Reply:
Whoops- conformance spec was supposed to be under "future work" (still in scope but probably not for a while)